_posts/2018-01-22-threads.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+author: Santiago Doldán
+title: Introduce threads to Suttna
+image: posts/threads/threads.png
+categories: slack teams threads bot suttna
+---
+
+![App](/assets/posts/threads/threads.png)
+
+# We are really proud to announce that Suttna now supports threads 😌
+
+When we started building Suttna, our main goal was to let users configure everything without leaving their messaging client.
+
+
+Having that in mind, our first idea was to create a subset of dialogs where users would be able to just start a conversation running
+a command (e.g. `@suttna configure`) and Suttna will guide them according to what they wanted to do.
+
+<br />
+
+![Excelent Idea](/assets/posts/threads/excelent-idea.jpg)
+
+<br />
+
+But we forgot something very important ... What happens with ALL the other users that are part of the channel but are not talking with Suttna? they might be asking
+
+### Why should I get 20 message notifications if I'm not even configuring a checkin, right? 🤯
+
+Now that Suttna has a few months running checkins for companies around the globe, we understood that bot conversations are nice, buuut that they can be a bit annoying (just a bit 😅).
+
+So we started collecting feedback from several accounts and decided that it was time to fix it.
+
+Most users asked for a new config option to prevent Suttna messaging the channel right after a user finishes answering their questions,
+but that was a solution for only one scenario, what about `@suttna configure`? that would still trigger a message notification for each new message.
+
+<br />
+
+![Thinking](/assets/posts/threads/thinking.jpg)
+
+<br />
+
+After thinking about other solutions, we decided to use `threads` (both messaging clients *Slack and MS Teams* support them). Suttna will create a new thread when
+entering a command and continue the conversation right there. This new flow won't generate message notifications and won't send too many messages to the group channel,
+much better of course.
+
+But it wasn't all a bed of roses, the Slack connector ([botbuilder-slack](https://github.com/suttna/botbuilder-slack)) that we've built for
+**Microsoft Botbuilder** didn't support threads, obviously that didn't last too much, one week after our connector supports them 😍.
+
+That's pretty much the story of why we added threads support, we would love to know from our users if this change makes the UX better or not, *we hope it does*.
```

Comment:
UX = user experience, evitaría la sigla. 

El final capaz que lo pensaría algo como, we love to improve constantly, and your opinion really matter to us! (threads wasn't an exception). Please don't hesitate and write us to . bla bla. Algo así?

Reply:
Me gusta la que tiro Ivan !